vmm/src/lib.rs

```diff
@@ -1135,6 +1145,11 @@ impl Vmm {
         Ok(true)
     }
 
+    fn can_increase_autoconverge_step(s: &MigrationState) -> bool {
+        let iteration = s.iteration.saturating_sub(AUTO_CONVERGE_ITERATION_DELAY);
+        iteration % AUTO_CONVERGE_ITERATION_INCREASE == 0
```

Comment:
This will return `true` for the first `AUTO_CONVERGE_ITERATION_DELAY` iterations (because of saturating sub resulting in `0`), which I believe is not what you want?

Reply:
ah yes, I think you are right. I want the first `AUTO_CONVERGE_ITERATION_DELAY` to be skipped